internal/pkg/bootstrap/container/database.go

```diff
@@ -16,7 +16,8 @@ package container
 
 import (
 	"github.com/edgexfoundry/edgex-go/internal/pkg/db/interfaces"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/di"
+
+	"github.com/edgexfoundry/go-mod-bootstrap/di"
```

Comment:
So DB boot strapping isn't moved to go-mod-bootstrap since it is service specific, correct?

Reply:
That's correct.

Reply:
What about changing the package name  to something like /pkg/database/container?

Reply:
Maybe but I see that as out of scope for this PR.

Reply:
Why? Isn't this about refactoring out the bootstrap package? Seems strange to leave something in it that really isn't bootstrap....

Reply:
It is bootstrap-related.  Just happens to be specific to edgex-go.  This file contains the helper functions related to the edgex-go-specific database handler (which is also under internal/pkg/bootstrap).  It makes sense to me to keep the same pattern in edgex-go for the service-specific bootstrap handlers that we're using in go-mod-bootstrap for the globally common bootstrap handlers.

Reply:
Ok, good with how it is now.